pkg/monitoring/perfscale/perfscale_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package perfscale_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestPerfscale(t *testing.T) {
```

Comment:
@davidvossel this needs to be changed to

```
func TestPerfscale(t *testing.T) {
   testutils.KubeVirtTestSuiteSetup(t)
}
```

to make the unittests pass.

Reply:
yup done, thanks